libmscore/edit.cpp

```diff
@@ -3609,7 +3613,7 @@ void Score::undoChangeFretting(Note* note, int pitch, int string, int fret, int
 //   undoChangeKeySig
 //---------------------------------------------------------
 
-void Score::undoChangeKeySig(Staff* ostaff, const Fraction& tick, KeySigEvent key)
+void Score::undoChangeKeySig(Staff* ostaff, const Fraction& tick, KeySigEvent key, InstrumentChange* ic)
```

Comment:
it's worth adding a note here to make it easier to understand this function's behaviour, like so:

    ..., InstrumentChange* ic /* = nullptr */)

instead of just 

    ..., InstrumentChange* ic)

Reply:
I see the logic, but I personally wouldn't bother. If you add another parameter in the header file or refractor the function name then the comment will most likely disappear without you realising.

__Pro tip:__ If you are in Qt Creator you can click on the function name in the `.cpp` file and press F2 to be taken straight to the declaration in the `.h` file (see [keyboard shortcuts](https://doc.qt.io/qtcreator/creator-keyboard-shortcuts.html): "follow symbol under cursor"). Other IDEs probably have a similar shortcut.